.github/workflows/release_examples.yml

```diff
@@ -16,53 +20,41 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: "0"
-      - uses: actions/cache@v1
-        with:
-          path: /tmp/release-snacks-${{ matrix.directory }}/
-          key: release-snacks-${{ matrix.directory }}-${{ github.sha }}
       - name: Serialize Examples
         env:
           REGISTRY: "ghcr.io/flyteorg"
         run: |
           REGISTRY=ghcr.io/flyteorg VERSION=latest make -C cookbook/${{ matrix.directory }} serialize
-      - name: Cache Serialized protos
+      - name: Archive Serialized protos
         env:
           REGISTRY: "ghcr.io/flyteorg"
         run: |
-          mkdir -p cookbook/release-snacks /tmp/release-snacks-${{ matrix.directory }}/
+          mkdir -p cookbook/release-snacks
           make -C cookbook archive
-          cp -r cookbook/release-snacks/ /tmp/release-snacks-${{ matrix.directory }}/*
-  releaser:
-    name: Release example
+      - name: Run GoReleaser
+        if: github.event_name != 'pull_request'
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.FLYTE_BOT_PAT }}
+
+  release-manifest:
+    name: Release Manifest
+    if: github.event_name != 'pull_request'
     runs-on: ubuntu-latest
-    needs: [ serialize-example ]
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: "0"
-      - uses: actions/cache@v1
-        with:
-          path: /tmp/release-snacks-core/
-          key: release-snacks-core-${{ github.sha }}
-      - uses: actions/cache@v1
-        with:
-          path: /tmp/release-snacks-case_studies/
-          key: release-snacks-case_studies-${{ github.sha }}
-      - uses: actions/cache@v1
-        with:
-          path: /tmp/release-snacks-integrations/
-          key: release-snacks-integrations-${{ github.sha }}
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: "0"
-          ref: ${{ steps.bump-version.outputs.tag }}
-      - name: Copy artifacts
+      - name: Serialize Examples
+        env:
+          REGISTRY: "ghcr.io/flyteorg"
         run: |
           mkdir -p cookbook/release-snacks
-          cp -r /tmp/release-snacks-core/* cookbook/release-snacks/
-          cp -r /tmp/release-snacks-case_studies/* cookbook/release-snacks/
-          cp -r /tmp/release-snacks-integrations/* cookbook/release-snacks/
-      - name: Run GoReleaser
+          cp cookbook/flyte_tests_manifest.json cookbook/release-snacks/flyte_tests_manifest.json
+      - name: Release test manifest
```

Comment:
I think go releaser should run once... we don't want two releases created, we want just one with all the files... I don't know if it has a mode where it can add to an existing release instead of failing if the release exists...

Reply:
I tested it, It will only create a release if not available. And it will have conflict we resource exist not else it will work fine